lib/nostrum/cache/user_cache.ex

```diff
@@ -2,22 +2,36 @@ defmodule Nostrum.Cache.UserCache do
   @default_cache_implementation Nostrum.Cache.UserCache.ETS
   @moduledoc """
   Cache behaviour & dispatcher for users.
+
+  You can call the functions provided by this module independent of which cache
+  is configured, and it will dispatch to the configured cache implementation.
+
+  By default, #{@default_cache_implementation} will be used for caching guilds.
+  You can override this in the `:caches` option of the `:nostrum` application
+  by setting the `:users` field to a different module implementing the
+  `Nostrum.Cache.UserCache` behaviour.
+
+  See the documentation for the `Nostrum.Cache.GuildCache` module for more details.
```

Comment:
Doesn't this belong into the `Nostrum.Cache.UserCache.ETS` module, with the typo fixed?